assets/templates/main.tmpl

```diff
@@ -25,12 +35,26 @@
     {{ else }}
       <link rel="stylesheet" href="{{ .PatternLibraryAssetsPath }}/css/main.css">
     {{ end }}
+    
     {{ if .HasJSONLD -}}
       {{ template "partials/json-ld/base" . }}
     {{- end }}
+
+    {{ if eq .Type "search"}}
+
+      {{/* if search a/b testing is at 100% then noindex tag else canonical tag */}}
```

Comment:
```suggestion
      {{ /* if search a/b testing is at 100% then noindex tag else canonical tag */ }}
```

Reply:
In this case, this is correct otherwise it will start complaining - https://learn.hashicorp.com/tutorials/nomad/go-template-syntax#comments

Reply:
interesting, all good then